apps/server/migrations/0005_model.ts

```diff
@@ -3,11 +3,15 @@ import { type Kysely, sql } from "kysely";
 export async function up(db: Kysely<unknown>): Promise<void> {
   await db.schema
     .createTable("model")
-    .addColumn("id", "text", (col) => col.primaryKey())
-    .addColumn("name", "text", (col) => col.notNull())
     .addColumn("workspace_id", "text", (col) =>
       col.notNull().references("workspace.id").onDelete("cascade"),
     )
+    .addColumn("family_id", "text", (col) =>
+      col.notNull().references("family.id").onDelete("cascade"),
+    )
+    .addColumn("id", "text", (col) => col.primaryKey())
+    .addColumn("name", "text", (col) => col.notNull())
```

Comment:
Just to also make sure, for a part like "(Ex: PCB-CTR065-001)", part number will just store the integer 1 right? And since we call it a number we assume it is a positive integer?

Reply:
hum, in this case model, which we want to rename to part, is not a concrete instance, does not seem to make sense to have a part number?

Reply:
Yeah, we need a `Root identifier for a particular part design or material` that the user can set.
- We could use the `id` column other wise ?